docs/apache-airflow-providers-mysql/index.rst

```diff
@@ -93,7 +93,7 @@ PIP requirements
 PIP package                 Version required
 ==========================  ======================
 ``mysql-connector-python``  ``>=8.0.11, <=8.0.22``
-``mysqlclient``             ``>=1.3.6,<1.4``
+``mysqlclient``             ``>=1.3.6,<3``
```

Comment:
I think the package versions at this doc are auto generated when provider is released and shouldn't be update manually. @potiuk is that correct?

Reply:
I think they are, but I also don't think there's any harm in editing it as part of this PR.